docker/cyclecloud8/scripts/cyclecloud_install.py

```diff
@@ -638,6 +638,10 @@ def main():
                         dest="storageManagedIdentity",
                         default=None,
                         help="Use a specified Managed Identity for storage access from the compute nodes")
+    parser.add_argument("--notGenerateCsConfig",
```

Comment:
Let's make this a boolean switch - setting "--generateCSConfig" means "create the file"
we can make handle the default-to-true in the run script